src/odemis/util/angleres.py

```diff
@@ -161,6 +160,16 @@ def _FindAngle(x_array, y_array, pixel_size, parabola_f):
     return theta, phi, omega
 
 
+def _flipDataIfMirrorFlipped(data):
+    # invert data and adjust metadata for inverted mirror
+    data = data[::-1, :]
+    data.metadata = data.metadata.copy()
+    data.metadata[model.MD_AR_FOCUS_DISTANCE] *= -1  # invert the focus distance for inverted mirror
+    # put new y pole coordinate
+    arpole = data.metadata[model.MD_AR_POLE]
+    data.metadata[model.MD_AR_POLE] = (arpole[0], data.shape[0] - 1 - arpole[1])
+
```

Comment:
You need to *return* data. Otherwise, this has no effect. In general, anyway, it's nicer to have the input and output arguments decoupled.